examples/grafana/README.md

```diff
@@ -0,0 +1,30 @@
+# grafana-ocp
+
+This template creates a custom Grafana instance preconfigured to gather Prometheus openshift metrics.
+It is uses OAuth token to login openshift Prometheus.
+
+
+## To deploy grafana
+Note: make sure to have openshift prometheus deployed.
+(https://github.com/openshift/origin/tree/master/examples/prometheus)
+
+1. ```oc create namespace grafana```
+2. ```oc new-app -f grafana-ocp.yaml```
+3. grab the grafana url ``` oc get route |awk 'NR==2 {print $2}' ```
+4. grab the ocp token, from openshift master run: ```oc sa get-token prometheus -n kube-system```
+5. browse to grafana via browser and add new prometheus DS to grafana.
```

Comment:
- change DS to datasource (could be confused with DaemonSet).
- mention that the data source URL is the prometheus route, and put an example URL in here

```https://prometheus-kube-system.apps.example.com```